README.md

```diff
@@ -100,37 +100,40 @@ This happens because these Django classes do not support [`__class_getitem__`](h
 
 1. You can go with our [`django_stubs_ext`](https://github.com/typeddjango/django-stubs/tree/master/django_stubs_ext) helper, that patches all the types we use as Generic in django.
 
-Install it:
+   Install it:
 
-```bash
-pip install django-stubs-ext  # as a production dependency
-```
+   ```bash
+   pip install django-stubs-ext  # as a production dependency
+   ```
 
-And then place in your top-level settings:
+   And then place in your top-level settings:
 
-```python
-import django_stubs_ext
-
-django_stubs_ext.monkeypatch()
-```
+   ```python
+   import django_stubs_ext
+   
```

Comment:
Trailing whitespace on this line broke CI.

- #1084

Reply:
good catch!